content/departments/cloud/technical-docs/v1.1/mi1-1_upgrade_process.md

```diff
@@ -38,12 +38,20 @@ GOBIN=~/.bin make install
 
 ## Steps
 
+### Create the instance upgrade tracking issue
+
+Open the output link in your browser to create the tracking issue
+
+```sh
+mi create-tracking-issue -target $VERSION // e.g. 4.0.0
```

Comment:
nit: if we need to reuse this `$VERSION` multiple times in a series of operations, I think it is much cleaner to have a line of bash like `export VERSION=<the version>`, then all commands refer to the `$VERSION` should just work.